src/stock-items/add-stock-item/stock-item-rules/add-stock-rules.component.tsx

```diff
@@ -217,12 +229,12 @@ const StockRulesAddOrUpdate: React.FC<AddStockRuleProps> = ({
                     value=""
                     text={t("chooseLocation", "Choose the location")}
                   />
-                  {locations?.map((location) => {
+                  {stockLocations?.map((location) => {
                     return (
                       <SelectItem
-                        key={location.uuid}
-                        value={location.uuid}
-                        text={location.display}
+                        key={location.id}
+                        value={location.id}
```

Comment:
Why are we using id instead of uuid as value? Does the endpoint support that? I would prefer we use uuids this is the best practice

Reply:
the endpoint am using returns such

Reply:
we should standardize the use of uuid instead of id. Should we add a TODO for that?

Reply:
noted

Reply:
@ojwanganto and @makombe the endpoint is a fhir endpoint and it has id in UUID formats
example ```${fhirBaseUrl}/Location?_summary=data&_tag=main store,main pharmacy,dispensary ```